From d2de18c05f2276a955c6925cbb1764110bc9b4ff Mon Sep 17 00:00:00 2001
From: Yu Chien Peter Lin <peterlin@andestech.com>
Date: Tue, 16 Apr 2024 14:56:57 +0800
Subject: [PATCH 7/8] qemu-riscv64_spl_defconfig: Enlarge malloc pool

Adjust CONFIG_SPL_SYS_MALLOC_SIZE to fix the boot failure:

  U-Boot SPL 2024.04-rc3-00075-gdb66fe3fa3-dirty (Apr 16 2024 - 14:55:40 +0800)
  Trying to boot from RAM
  Could not get FIT buffer of 1482152 bytes
          check CONFIG_SPL_SYS_MALLOC_SIZE

Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
---
 configs/qemu-riscv64_spl_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/qemu-riscv64_spl_defconfig b/configs/qemu-riscv64_spl_defconfig
index 47347d529c..37d46cedbb 100644
--- a/configs/qemu-riscv64_spl_defconfig
+++ b/configs/qemu-riscv64_spl_defconfig
@@ -19,6 +19,7 @@ CONFIG_DISPLAY_CPUINFO=y
 CONFIG_DISPLAY_BOARDINFO=y
 CONFIG_SPL_MAX_SIZE=0x100000
 CONFIG_SPL_SYS_MALLOC=y
+CONFIG_SPL_SYS_MALLOC_SIZE=0x0200000
 CONFIG_SPL_OPTEE_IMAGE_LOAD_ADDR=0xf1000000
 # CONFIG_CMD_MII is not set
 CONFIG_SYS_RELOC_GD_ENV_ADDR=y
-- 
2.34.1

